src/app/zap-templates/zcl/data-model/chip/global-attributes.xml

```diff
@@ -19,7 +19,7 @@ limitations under the License.
     <attribute side="client" code="0xFFFD" define="CLUSTER_REVISION_CLIENT" type="int16u"   default="1"                 >ClusterRevision</attribute>
     <attribute side="server" code="0xFFFD" define="CLUSTER_REVISION_SERVER" type="int16u"   default="1"                 >ClusterRevision</attribute>
     <attribute side="client" code="0xFFFC" define="FEATURE_MAP_CLIENT"      type="bitmap32" default="0" optional="true">FeatureMap</attribute>
```

Comment:
Should `optional="true"` also be removed from here?

Reply:
I believe yes according to spec: feature map seems required.

Reply:
I am referring to line 21 here not line 22 where I have already removed it

Reply:
Ah, client side? I would copy what revision did, which seems to have modified both. For consistency we should probably modify both for feature map too.